src/Hazelcast.Net/Serialization/Compact/CompactWriterExtensions.GeneratedCode.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#nullable enable
+
+using System;
+using Hazelcast.Models;
+
+namespace Hazelcast.Serialization.Compact;
+
+internal static class CompactWriterExtensions
+{
+    public static void WriteAny(this ICompactWriter writer, string fieldname, FieldKind kind, object? value)
+    {
+        switch (kind)
+        {
+            // <generated>
+
+            case FieldKind.Boolean: writer.WriteBoolean(fieldname, (bool) value); break;
```

Comment:
Do you know why github gave a lot of warnings in these lines about "Unboxing a possibly null value"?